source/pymongo.txt

```diff
@@ -76,10 +75,24 @@ Connect to MongoDB Atlas
 
    import pymongo
 
-   client = pymongo.MongoClient(
-      "mongodb+srv://<username>:<password>@<cluster-url>/test?retryWrites=true&w=majority")
-   db = client.test
+   // replace this with your MongoDB connection string
+   conn_str = "<your MongoDB Atlas connection string>"
+
+   client = pymongo.MongoClient(conn_str, serverSelectionTimeoutMS=5000)
+
+   try:
+       print(client.server_info())
+   except Exception:
+       print("Unable to connect to the server.")
+
+If the connection succeeds before a five-second timeout, you should see a
```

Comment:
```suggestion
If the connection succeeds before a five-second timeout, you see a
```
Nit: removing "should" makes the outcome sound more concrete opposed to saying "you might see this"

Also applies to Motor page.

Reply:
I'm undecided on this one. I think "should" can make it sound less definitive as you mention which would be more appropriate for the case in which something else can possibly happen, but  "you see" sounds awkward and informal in general. Maybe it needs to be "you will see" to describe the future.